credentials/src/main/kotlin/SSI.kt

```diff
@@ -105,11 +113,9 @@ public data class CreateVcOptions(
 )
 
 public data class CreateVpOptions(
-  // TODO: Add this
-  // val presentationDefinition: PresentationDefinitionV2,
-  // val verifiableCredentialJwts: List<String>,
-  // TODO: remove this for verifiableCredentialJwts instead
-  val verifiableCredentials: List<VerifiableCredentialType>,
+  val presentationDefinition: PresentationDefinitionV2,
+  val verifiableCredentialJwts: List<String>,
```

Comment:
Using our own types let's us evolve this more easily. We can additionally provide APIs for easy creation of VcJwt to and from a String / JWS / JWT.

```suggestion
  val verifiableCredentialJwts: List<VcJwt>,
```

Reply:
Yup good call